recon/src/main/java/org/hps/recon/filtering/EventFlagFilter.java

```diff
@@ -10,8 +11,9 @@
 public class EventFlagFilter extends EventReconFilter {
 
     private static Logger LOGGER = Logger.getLogger(EventFlagFilter.class.getPackage().getName());
-    
-    String[] flagNames = {"svt_bias_good", "svt_position_good", "svt_burstmode_noise_good", "svt_event_header_good", "svt_latency_good", "svt_readout_overlap_good"};
+    private static Level LEVEL = Level.WARNING;
+
+    String[] flagNames = {"svt_bias_good", "svt_position_good", "svt_burstmode_noise_good", "svt_event_header_good", "svt_latency_good"};
```

Comment:
I think this will break the apv25 readout crosstalk filter.

Reply:
It doesn't look like it does as this gets overwritten by the steering file. Is there a reason for removing "svt_readout_overlap_good" from this here though? I would prefer we just keep it so a list of all possible flags is kept here.

Reply:
This should be fixed now. I reverted the flag names array.